tests/resources/telescope_positions-North-utm-withoutmeta.ecsv

```diff
@@ -0,0 +1,54 @@
+# %ECSV 1.0
+# ---
+# datatype:
+# - {name: asset_code, datatype: string, description: operational asset code}
+# - {name: sequence_number, datatype: string, description: sequence number}
+# - {name: utm_east, unit: m, datatype: float64}
+# - {name: utm_north, unit: m, datatype: float64}
+# - {name: alt, unit: m, datatype: float64, description: telescope position upwards relative to NNN}
+# - {name: geo_code, datatype: string, description: Geographical Position Code}
+# meta: !!omap
+# - data_type: position_file
+# - description: "CTAO Northern Array Element Coordinates"
+# - center_northing: "3185067.275 m"
+# - center_easting: "217608.975 m"
+# - center_alt: "2177.0 m"
+# - corsika_obs_level: "2156 m"
+# - corsika_sphere_radius: {LST: 12.5 m, MST: 9.15 m, SST: 3 m}
+# - corsika_sphere_center: {LST: 16 m, MST: 9 m, SST: 3.25 m}
+# - EPSG: 32628
+# - comments: |-
+#       following CTA-SPE-PSC-000000-0001 1c
+#       Subset of positions defined in reference repository
+#       https://gitlab.cta-observatory.org/cta-science/array-element-positions/-/tree/main
+#       Array center coordinates defined as centre of the 4 LST
+#       position (this is slightly different to the definition
+#       in CTA-SPE-PSC-000000-0001 1c)
+#       Never change positions here. This should always be
+#       synchronized with the reference repository
```

Comment:
There is still a metadata, just not as defined by the schema, though I am not sure this comment makes a difference.